modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/dto/v1_0/converter/UserGroupResourceDTOConverter.java

```diff
@@ -61,10 +66,20 @@ public com.liferay.headless.admin.user.dto.v1_0.UserGroup toDTO(
 		return new com.liferay.headless.admin.user.dto.v1_0.UserGroup() {
 			{
 				setActions(dtoConverterContext::getActions);
+				setCreator(
+					() -> CreatorUtil.toCreator(
+						_portal,
+						_userLocalService.fetchUser(userGroup.getUserId())));
```

Comment:
we need to export these two new fields only if are required in the nestedFields
keep this as a code suggestion on how to do it
```
() -> NestedFieldsSupplier.<Integer>supply(
						"taxonomyCategoryUsageCount",
						fieldName -> {
```